src/main/kotlin/com/example/noswimmingserver/domain/auth/service/QueryGoogleAuthLinkService.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.noswimmingserver.domain.auth.service
+
+import com.example.noswimmingserver.domain.common_user.presentation.dto.response.LinkResponse
+import com.example.noswimmingserver.infra.feign.properties.GoogleFeignProperties
+import org.springframework.stereotype.Service
+
+@Service
+class QueryGoogleAuthLinkService(
+    private val googleFeignProperties: GoogleFeignProperties,
+) {
+
+    companion object {
+        private const val GOOGLE_LINK = "%s?client_id=%s&redirect_uri=%s&response_type=code" +
+                "&scope=https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/userinfo.profile"
```

Comment:
🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (16) (should be 12)